### PR TITLE
Clearer instructions for Desktop install docs

### PR DESCRIPTION
**Pull request by mvitale1989**
#### Summary

This PR addresses some clarity issues with the Desktop app's installation instructions for generic Linux distributions, and explicitly mentions what are the steps for installing the AppImage by referencing the official docs.

#### Ticket Link

https://github.com/mattermost/docs/issues/7230

### CODE REVIEW

**Review by amyblais — APPROVED** [review]

File: source/collaborate/install-desktop-app.rst [hunk 1]
```diff
@@ -168,7 +168,11 @@ You can download the `desktop app <https://mattermost.com/apps/>`_ directly from
 
 .. tab:: Generic Linux
 
-  A AppImage distribution (Beta) of a compressed tarball is available. Automatic app updates are supported and enabled. When a new version of the desktop app is released, your app updates automatically. 
+  The Desktop app is available in two formats which are usable on most Linux distributions: a compressed tarball, and an AppImage binary. Both can be downloaded from the `Desktop App's Github releases page <https://github.com/mattermost/desktop/releases>`. Automatic app updates are supported and enabled. When a new version of the desktop app is released, your app updates automatically.
```
Comment (cwarnermm) [inline source/collaborate/install-desktop-app.rst:171]:
```suggestion
  The Desktop app is available in two formats which are usable on most Linux distributions: a compressed tarball, and an AppImage binary. Both can be downloaded from the `Desktop App's Github releases page <https://github.com/mattermost/desktop/releases>`_. Automatic app updates are supported and enabled. When a new version of the desktop app is released, your app updates automatically.
```

File: source/collaborate/install-desktop-app.rst [hunk 2]
```diff
@@ -168,7 +168,11 @@ You can download the `desktop app <https://mattermost.com/apps/>`_ directly from
...
+
+  For instructions on how to use the AppImage binary, please refer to the  `AppImage Quickstart documentation page <https://docs.appimage.org/introduction/quickstart.html>`.
```
Comment (cwarnermm) [inline source/collaborate/install-desktop-app.rst:173]:
```suggestion
  For instructions on how to use the AppImage binary, please refer to the  `AppImage Quickstart documentation page <https://docs.appimage.org/introduction/quickstart.html>`_.
```

**Review by cwarnermm — APPROVED** [review]
Thank you so much for this docs PR, @mvitale1989!

File: source/collaborate/install-desktop-app.rst [hunk 3]
```diff
@@ -168,7 +168,11 @@ You can download the `desktop app <https://mattermost.com/apps/>`_ directly from
 
 .. tab:: Generic Linux
 
-  A AppImage distribution (Beta) of a compressed tarball is available. Automatic app updates are supported and enabled. When a new version of the desktop app is released, your app updates automatically. 
+  The Desktop app is available in two formats which are usable on most Linux distributions: a compressed tarball, and an AppImage binary. Both can be downloaded from the `Desktop App's Github releases page <https://github.com/mattermost/desktop/releases>`_. Automatic app updates are supported and enabled. When a new version of the desktop app is released, your app updates automatically.
```
Comment (devinbinnie) [inline source/collaborate/install-desktop-app.rst:171]: @mvitale1989 @cwarnermm A point of clarity: Automatic updates are only supported on the AppImage build, not the `.tar.gz`

Comment (cwarnermm) [reply source/collaborate/install-desktop-app.rst:171]: Thanks, @devinbinnie - I'll clarify in a follow-on docs PR

**Pull request merged**